mep/common/templates/common/widgets/checkbox_fieldset.html

```diff
@@ -0,0 +1,18 @@
+{% with id=widget.attrs.id name=widget.attrs.name %}
```

Comment:
Please add a template comment documenting what this is / what it belongs with.  Should never be called directly, right? So no need to document what variables it expects.

Reply:
Right. Never called directly, only ever by the widget.